src/components/app-detail/app-detail.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div>
+    <v-card v-if="trackDetail.trackName" dark>
+      <v-card-text class="d-flex flex-no-wrap justify-space-between">
+        <div>
+          <v-card-title class="headline white--text">Detail</v-card-title>
+        </div>
+      </v-card-text>
+      <v-container>
+        <v-row dense>
+          <v-col cols="3">
+            <v-avatar class="ma-3" size="125" tile>
+              <img :src="trackDetail.artworkUrl100" />
+            </v-avatar>
+          </v-col>
+          <v-col cols="9">
+            <div>
+              <h2>
+                <span>Name:</span>
+                <span class="font-weight-light"> {{ trackDetail.trackName }}</span>
+              </h2>
+              <h2>
+                <span>Artist:</span>
+                <span> {{ trackDetail.artistName }}</span>
+              </h2>
+              <h2>
+                <span>Collection:</span>
+                <span class="font-weight-light"> {{ trackDetail.collectionName }}</span>
+              </h2>
+              <h2>
+                <span>Country:</span>
+                <span class="font-weight-light"> {{ trackDetail.country }}</span>
+              </h2>
+              <h2>
+                <span>Genre:</span>
+                <span class="font-weight-light"> {{ trackDetail.primaryGenreName }}</span>
+              </h2>
+            </div>
+          </v-col>
+        </v-row>
+      </v-container>
+    </v-card>
+    <v-btn text color="grey" :to="'/'" v-else>
```

Comment:
I suggest using anchor tag for semantic navigation

Reply:
<router-link> is intended for in-app links (e.g., to a page within your app). Use <a> for external links.